mne_rsa/viz.py

```diff
@@ -67,3 +69,168 @@ def plot_dsms(dsms, names=None, items=None, n_rows=1, cmap='viridis',
     if title is not None:
         plt.suptitle(title)
     return fig
+
+
+def _click_func(ax, ch_idx, dsms, cmap):
+    """ Function used to plot a single DSM interactively.
+
+    Parameters
+    ----------
+    ax: matplotlib.Axes.axes
+        Axes.axes object on which a new single DSM is plotted.
+    ch_idx: int
+        Index of a channel.
+    dsms: ndarray, shape (n_sensors, n_dsm_datapoint)
+        DSMs of MEG recordings; there's one DSM for each sensor.
+    cmap: str
+        Colormap used for plotting DSMs.
+        Check matplotlib.pyplot.imshow for details.
+    """
+    dsm = dsms[ch_idx]
+    dsm = distance.squareform(dsm)
+    ax.imshow(dsm, cmap=cmap)
+
+
+def _plot_dsms_topo(dsms, info, layout=None, fig=None, title=None,
```

Comment:
I don't like the name `plot_dsms_topos` later on, let's rename this function first:

```suggestion
def _plot_dsms_topo_timepoint(dsms, info, layout=None, fig=None, title=None,
```